staff/AbelusM/css-layout/main.css

```diff
@@ -0,0 +1,35 @@
+* {
+    padding: 0;
+    margin: 0;
+ }
+ 
+ html,
+ body {
+    min-height: 100% !important;
```

Comment:
yew! no entiendo bien el porqué de mi-height y el !important. Que es lo que hace exactamente?

Reply:
is just for avoid to other stuff ever change the minimum height of the html or the body. the thing is !important makes sure of that instruction always will be the main option. 

es para tratar de evitar que cambie la altura mínima, el !important hace que eso siempre prevalezca para evitar modificarlo con otras instrucciones posteriores.

Reply:
NOT required, trying removing it ,)

Reply:
NOT required

Reply:
Agree , totally not necessary